### PR TITLE
[ws-manager-mk2] Remove persistent volume claim feature 

### PR DESCRIPTION
**Pull request by Furisto**
## Related Issue(s)

Fixes WKS-212

#### Preview status

gitpod:summary

## Build Options

<details>
<summary>Build</summary>

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer</summary>

- [ ] analytics=segment
- [ ] with-dedicated-emulation
- [x] with-ws-manager-mk2
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment</summary>

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
</details>
 
/hold

### CODE REVIEW

File: install/installer/pkg/config/v1/config.go [hunk 1]
```diff
@@ -337,28 +331,11 @@ type WorkspaceTemplates struct {
...
 	Runtime   WorkspaceRuntime    `json:"runtime" validate:"required"`
 	Resources Resources           `json:"resources" validate:"required"`
 	Templates *WorkspaceTemplates `json:"templates,omitempty"`
 
-	// PrebuildPVC is the struct that describes how to setup persistent volume claim for prebuild workspace
-	PrebuildPVC PersistentVolumeClaim `json:"prebuildPVC" validate:"required"`
-
-	// PVC is the struct that describes how to setup persistent volume claim for regular workspace
-	PVC PersistentVolumeClaim `json:"pvc" validate:"required"`
-
```
Comment (WVerlaek) [inline install/installer/pkg/config/v1/config.go:361]: note: need to clean up the config here as well: https://github.com/gitpod-io/ops/blob/main/deploy/workspace/installer-values.yaml

**Review by WVerlaek — COMMENTED** [review]
lgtm, could you run the integration tests as well?

**Comment by Furisto** [comment]
> lgtm, could you run the integration tests as well?
I ran the prebuild tests from my workspace. Tried to run the other ones thorugh CI as well but my preview installation got so messed up during installation that I had to recreate the PR. Prebuild ones are the only ones that should really be affected though.

**Comment by WVerlaek** [comment]
@Furisto needs rebasing

**Review by WVerlaek — APPROVED** [review]

**Review by akosyakov — APPROVED** [review]
Should we do a smoke test before merging?

**Comment by Furisto** [comment]
> Should we do a smoke test before merging?

I already did this in the preview environment that I created from my workspace successfully but the setup of the environment in the CI got messed up (unrelated to these changes), so I removed it from the PR.

**Pull request merged**